docs/installation/presto-admin-configuration.rst

```diff
@@ -3,11 +3,23 @@
 ==========================
 Presto-Admin Configuration
 ==========================
-A Presto cluster consists of a coordinator node and one or more workers nodes. A coordinator and worker may be located on the same node, meaning that you can have a single-node installation of Presto, but having a dedicated node for the coordinator is recommended for better performance, especially on larger clusters.
-
-In order to use ``presto-admin`` to manage software on a cluster of nodes, you must specify a configuration for ``presto-admin``. This configuration indicates the nodes on which to install as well as other credentials.
-
-To set up a configuration, create a file ``/etc/opt/prestoadmin/config.json`` with the content below. Replace the variables denoted with brackets <> with actual values enclosed in quotations. The user specified by ``username`` must have sudo access on all of Presto nodes, and ``presto-admin`` also must be able to login to all of the nodes via SSH as that user (see :ref:`ssh-configuration-label` for details on how to set that up). The file should be owned by root with R/W permissions (i.e. 622).
+A Presto cluster consists of a coordinator node and one or more workers nodes.
```

Comment:
The format change isn't needed for anything.  It just makes it easier to read.  Single newlines in markdown are basically equivalent to spaces.  All of the lines will together still be considered one paragraph for html purposes.  Whoever initially wrote this doc didn't realize that and so put everything that was supposed to be one paragraph on a single super long line.